app.js

```diff
@@ -1,13 +1,25 @@
 const express = require("express");
 const { getTopics } = require("./controllers/topics-controllers");
+const { getArticleById } = require("./controllers/articles-controllers");
 const { getApi } = require("./controllers/api-controllers");
 
 const app = express();
 
-app.use = (express.json())
+app.use(express.json());
 
 app.get("/api/topics", getTopics)
 
+app.get(`/api/articles/:article_id`, getArticleById)
+
+app.use((err, req, res, next) => {
+    if (err.status) {
+      res.status(err.status).send({ msg: err.msg });
+    } else if (err.code === "22P02") {
```

Comment:
Should extract error handling logic out into separate functions (see the last step of the notes on Error Handling)